brownfield_django/templates/main/team/team_home.html

```diff
@@ -41,7 +44,7 @@
                 <a class="btn btn-default pull-right" href=http://brownfieldref.ccnmtl.columbia.edu/" target="_blank">View</a>
                 </li>
               {% else %}
-                <a class="btn btn-default pull-right" href="../../../media/{{ document.link }}" target="_blank">View</a>
+                <a class="btn btn-default pull-right" href="{% static '/media/{{ document.link }}" target="_blank">View</a>
```

Comment:
Also, is that '/media/' extraneous? the static url usually includes that part. Or is there a '/media/media/' structure on this project?